src/app/donations/donations.js

```diff
@@ -0,0 +1,59 @@
+angular.module('prx.donations', ['prx.bus', 'angulartics',])
+.run(function (Bus, $analytics) {
+  var category = 'Outbound';
+
+  Bus.on('donate.outbound', function (account, url) {
+    $analytics.eventTrack('Donate', {
+      category: category,
+      label: 'Account-' + account.id.toString() + '-' + url,
+      // hitcallback: function () {
+      //   $window.location.href = url;
+      // }
+    });
+  });
+})
+.service('prxDonateURL', function () {
+  this.forAccount = function (account) {
+    var map = {
+      '45139': 'http://themoth.org/support?utm_source=PRX&utm_medium=Donate%20Button&utm_campaign=PRX%20-%20Moth%20-%20Donate',
+      '103576': 'http://thetruthpodcast.com/Support.html',
+      '551': 'http://99percentinvisible.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '12248': 'http://loveandradio.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '123677': 'http://www.storycentral.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '31': 'http://toe.prx.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '561': 'http://www.radiodiaries.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '579': 'http://www.kitchensisters.org/support/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '179116': 'http://www.theheartradio.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+    };
+
+    return map[account.id.toString()];
+  };
+})
+.directive('prxDonate', function(prxDonateURL, $analytics, $window, $timeout, Bus) {
+  return {
+    restrict: 'E',
+    replace: true,
+    template: '<div class="donate">Donate</div>',
```

Comment:
Yeah I don't think it's really a problem except that ng-click also binds to touchstart because we are including the ngtouch package, and it means you have to worry less about the cleanup stuff.

I think what you have here is kosher, but if you are not using ng-click because you want to avoid making a controller, I can help you with that. I'd also sort of take issue with the idea that controllers are only useful for inter-directive communication. I know that the docs say that directive controllers is how you achieve inter-directive communications, but I don't think it's fair to suggest that's the only use (especially now that bindToController is available).

Let me know if you want me to help you through what I am talking about. Otherwise, I'm happy to merge what you have.